src/gui/editors/PianoRoll.cpp

```diff
@@ -1687,7 +1687,7 @@ void PianoRoll::mousePressEvent(QMouseEvent * me )
 		m_moveStartY = me->y();
 	}
 
-	if(me->button() == Qt::LeftButton &&
+	if( me->button() == Qt::LeftButton &&
```

Comment:
We do infact have our own [Coding Conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions), and so we try to be consistent with that rather than the existing codebase if we were to make changes. We would do something like this instead.
```suggestion
if (me->button() == Qt::LeftButton &&
```